docs/platform/self_hosting/configuration.mdx

```diff
@@ -129,61 +233,212 @@ tolgee.smtp.username=*****************
+```
+tolgee.socket-io.allowed-transports[0]=websocket
+tolgee.socket-io.allowed-transports[1]=polling
+```
+
+See [Spring's documentation](https://docs.spring.io/spring-boot/docs/2.6.4/reference/html/features.html#features.external-config.yaml.mapping-to-properties) for more information.
+:::
+
+## Cache
+Prefix: `tolgee.cache`
+
+At the expense of higher memory footprint, Tolgee can use a cache to reduce the stress on the database and fetch the
+data it needs faster. Cache is also used to track certain states, such as rate limits.
+
+### enabled
+Whether Tolgee should use a cache. (default: `false`)
+
+### use-redis
+Whether Tolgee should use Redis to store cache data instead of storing it in-memory. (default: `false`)
+
+:::info
+In a distributed environment, you **should** use a Redis server to ensure consistent enforcement of rate limits, as
+they heavily rely on cache. For a simple single-node deployment, in-memory cache is sufficient.
+:::
+
+:::info
+Tolgee uses [Redisson](https://github.com/redisson/redisson) to interface with the Redis server. You can find the
+properties Redisson expects [here](https://github.com/redisson/redisson/tree/56ea7f5/redisson-spring-boot-starter#2-add-settings-into-applicationsettings-file).
+:::
+
+### default-ttl
+TTL of cache data, in milliseconds. (default: `120 * 60 * 1000` = 2 hours)
+
+### caffeine-max-size
+Maximum size of the Caffeine cache. When exceeded, some entries will be purged from cache. Set to -1 to disable size
+limitation. This has no effect when Redis cache is used. See
+[Caffeine's documentation about size-based eviction](https://github.com/ben-manes/caffeine/wiki/Eviction#size-based)
+for more information. (default: `-1`)
+
+## Sentry
+Prefix: `tolgee.sentry`
+
+Tolgee uses [Sentry](https://sentry.io/) for error reporting.
+
+### server-dsn
+Server DSN. If unset, error reporting is disabled on the server. (default: `null`)
+
+### client-dsn
+Client DSN. If unset, error reporting is disabled on the client. (default: `null`)
+
+## Sendinblue
+Prefix: `tolgee.send-in-blue-properties`
+
+Configuration for the [Sendinblue](https://www.sendinblue.com/) marketting tool. Used by Tolgee Cloud.
+
+### api-key
+Sendinblue API key. (default: `null`)
+
+### list-id
+The marketting list ID users will be added/removed from. (default: `null`)
+
+## Import
+Prefix: `tolgee.import`
+
+Bulk-imports exported json files in the database during startup. Useful to quickly provision a development server, and
+used for unit testing.
+
+### dir
+File path of the directory where the file to import are located.
+
+### create-amplicit-api-key
+Whether an implicit API key should be created. (default: `false`)
+
+The key is built with a predictable format: `${lowercase filename (without extension)}-${initial username}-imported-project-implicit`
+
+:::danger
+While this is useful for unit tests, make sure to keep this **disabled** if you're importing projects on a production
```

Comment:
useful for tests

- It's not useful for unit test, but for end to end tests. Not sure if we have to document this.

Reply:
Right, e2e vs unit, I always do this dumb mistake haha. I think it's still good to document it, either for people who will want to tinker with Tolgee.